docs/tutorial.md

```diff
@@ -495,6 +495,44 @@ Your take really. Let's take the former approach:
 
 We're using [`#resolve()`](https://doc.esdoc.org/github.com/Kinto/kinto.js/class/src/collection.js~Collection.html#instance-method-resolve) to mark a conflict as resolved: it accepts a conflict object, and a resolution one; the latter is what will be updated locally and sent for resynchronization on a next call to [`#sync()`](https://doc.esdoc.org/github.com/Kinto/kinto.js/class/src/collection.js~Collection.html#instance-method-sync) — which is exactly what we're doing after we've resolved all our conflicts.
 
+## Receiving changes via Websocket
+
+`collection.sync()` only sync the data after user triggerred the sync. Very often we want to live sync user data among different clients. In that case, you'll need to listen to the changes via a websocket server.
```

Comment:
While explicitly calling `collection.sync()` works nicely, sometimes we want the synchronization operation to happen automatically in the background. In that case, we need to listen to change events sent from a [WebSocket](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API) server.